.github/workflows/github-actions-after-push.yml

```diff
@@ -2,7 +2,7 @@ name: GitHub Actions After Push Into Develop Branch
 on:
   push:
     branches:
-      - develop
+      - develop-ts
 jobs:
  cypress-run:
     runs-on: ubuntu-latest

```

Comment:
It looks like this script still doesn't work. Figure it out. Maybe you forgot to change your develop named branch in the necessary places.